### ISSUE TITLE
jdown does not convert Markdown to HTML after save (react-static)

### ISSUE THREAD

**Issue by babycourageous** [DanWebb/jdown#8]
Hi there

Very specific and bizarre issue I am coming up against with version `0.5.0` using `react-static` that doesn't seem to happen in the `0.1.0` version you use on your site. Basically on first compile with `yarn start` Markdown files are converted into HTML as expected. However subsequent saves to `config.static.js` or Markdown files results in the `json` containing raw Markdown. 

I discovered this and then tested with your site as is. The bug was not present. When I updated jdown on my local copy of your site to the latest version this behavior started. Here's a snippet of `privacy-policy` log on first run and then after I changed the `.md` file: 

Before: 
```
contents: 
    '<p>This page is used to inform website visitors of our policies
```
After
```
contents:
     '\nThis page is used to inform website visitors of our policies
```
The rest of the contents contain raw Markdown like `##` instead of `h2` etc...

I assume this is a `jdown` issue since it shows up after the updated version.
Hopefully this makes some sense - let me know if you have any ideas!
Thanks!

**Comment by DanWebb** [comment]
Hi @babycourageous thanks so much for bringing this to attention. I managed to replicate and apply a fix for the issue you were seeing in https://github.com/DanWebb/jdown/commit/6d24e8d5bc968037608f5e526df2a9d17558d27e If you upgrade to 0.5.1 it should all be updating correctly on file save.

If you're interested - The issue was due to some default options not applying the second time jdown was run, I always forget `Object.assign` isn't a deep clone 🙈so when properties are deleted that affects the original object (in this case the default options object) too which can end up causing strange issues like this.
